paper-ieee-trans.tex

```diff
@@ -346,23 +348,23 @@ \subsection{INtegrating Distributed data Infrastructures for Global
 \section{New trends in software reliability}
 \label{sec:ntsr}
 
-{\sl INDIGO--DataCloud} is the most recent project included in this paper thus it
+{\sl INDIGO--DataCloud} is the most recent project included in this paper thus, it
 emerged from the lessons learned in the previous experiences described in
 Section \ref{sec:ev}. The expertise gathered throughout these years was highly
 profitable in terms of the evaluation and application of innovative software engineering
 methodologies, the management of new technologies to put those methodologies
 into practice, and the compilation and analysis of the appropriate set of metrics to measure the
 success of the solutions implemented. All the above with the additional complexity of
-being framed into large collaborative projects with an extensive list of partners.
+large collaborative projects with an extensive list of partners.
```

Comment:
not sure if removing "being framed" the sentence makes sense..

Reply:
up to you